modules/oceanbase/src/main/java/org/testcontainers/oceanbase/OceanBaseCEContainerProvider.java

```diff
@@ -9,7 +9,7 @@
  */
 public class OceanBaseCEContainerProvider extends JdbcDatabaseContainerProvider {
 
-    private static final String DEFAULT_TAG = "4.2.2";
+    private static final String DEFAULT_TAG = "4.2.1-lts";
```

Comment:
We don't usually update those images but if I understand correctly this is more stable, right?

Reply:
Yes, it's the long term supported version, which should be more stable.